src/state/item/item-reducers.js

```diff
@@ -81,6 +81,7 @@ export default function (state = initialState, action) {
                 ...state,
                 activeItem: document,
                 activeItemEditing: document,
+                waitingForItems: false, // TODO: revisit
```

Comment:
For some reason, when we start to fetch a single item, it seems to set this value to `true`.

- [ ] Update this so that when fetching a single item -- instead of greying out the main list -- it greys out the metadata sidepanel if it is open.